### PR TITLE
refactor: clean unused networkConfig

### PR DESCRIPTION
**Pull request by satish-ravi**
### Description

Removed unused config, replaced config that could be easily inferred from another config.

### Test plan

CI

### Related issues

N/A

### Backwards compatibility

N/A

### Network scalability

N/A

### CODE REVIEW

File: src/utils/currencies.ts [hunk 1]
```diff
@@ -4,14 +4,14 @@ export enum Currency {
...
-// we use in address-metadata
+/**
+ * @deprecated Should use tokenIds / symbols from token list instead.
+ */
 export enum CiCoCurrency {
   CELO = 'CELO',
   cUSD = 'cUSD',
   cEUR = 'cEUR',
   cREAL = 'cREAL',
-  ETH = 'ETH',
```
Comment (satish-ravi) [inline src/utils/currencies.ts:14]: to make knip happy. This wasn't used anywhere. We should ultimately get rid of this, but there are some legacy fiat exchanges dependencies on this

File: secrets.json.enc
Comment (satish-ravi) [inline secrets.json.enc:1]: just removed RECAPTCHA_SITE_KEY here

**Review by kathaypacific — APPROVED** [review]
🧹🥳

**Review by jeanregisser — APPROVED** [review]
Awesome! 🚀

**Pull request merged**